client/src/components/Create.jsx

```diff
@@ -53,12 +57,17 @@ class Create extends React.Component {
         inviteEmails: this.state.inviteEmails
       }
     }).then(function(response) {
-      console.log('Create handleSubmit response', response);
+      that.setState({eventId: response.data.id, redirectToEvent: true});
+    }).catch(function(error) {
+      console.log(error);
     });
   }
 
   render() {
     // Set component title based on URL
+    if (this.state.redirectToEvent) {
```

Comment:
I _think_ you can use `if (this.state.eventId)` since the init value is `0` which is falsy and once `this.state.eventId` is updated by your axios post success callback with a value that is not `0`

Then you can remove `redirectToEvent` from the state and all references to it?